ui/app/templates/vault/cluster/access/oidc/assignments/assignment/details.hbs

```diff
@@ -1 +1,78 @@
-details
\ No newline at end of file
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs">
+      <ul>
+        <li>
+          <span class="sep">&#x0002f;</span>
+          <LinkTo @route="vault.cluster.access.oidc.assignments">
+            Assignments
+          </LinkTo>
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{@model.name}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<div class="tabs-container box is-sideless is-fullwidth is-paddingless is-marginless">
+  <nav class="tabs" aria-label="tabs">
+    <ul>
+      <LinkTo @route="vault.cluster.access.oidc.assignments.assignment.details" @model={{@model}}>
+        Details
+      </LinkTo>
+    </ul>
+  </nav>
+</div>
+
+<Toolbar>
+  <ToolbarActions>
+    {{#if @model.canDelete}}
+      <ConfirmAction
+        @buttonClasses="toolbar-link"
+        @onConfirmAction={{this.delete}}
+        @confirmTitle="Delete assignment?"
+        @confirmMessage="This assignment will be permanently deleted. You will not be able to recover it."
+        @confirmButtonText="Delete"
+      >
+        Delete assignment
+      </ConfirmAction>
+    {{/if}}
+    {{#if @model.canEdit}}
+      <ToolbarLink @params={{array "vault.cluster.access.oidc.assignments.assignment.edit" @model.name}}>
+        Edit assignment
+      </ToolbarLink>
+    {{/if}}
+  </ToolbarActions>
+</Toolbar>
+
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  <InfoTableRow @label="Name" @value={{@model.name}} />
+  <InfoTableRow
```

Comment:
Claire you and I talked about pulling InfoTableItemArray out of InfoTableRow however in the end we use some hbs formatting from InfoTableRow. All of the label and row formatting. It was easier and cleaner to keep it in that component. Also, it keeps with the standard that it's in an InfoTableRow, instead of some other stand alone component.

Reply:
Yeah - that makes sense! I noticed clicking 'view all' for this component takes you to the /list page for **all** entities or groups and it isn't filtered for this particular assignment. I asked @yankun91 and the preference would be to add the tabs in this design option: 
<img width="461" alt="Screen Shot 2022-08-01 at 3 12 31 PM" src="https://user-images.githubusercontent.com/68122737/182254543-b684e41b-13c2-4152-933a-412f69565937.png">

Another option (if it's easier) is to utilize the existing list view, and use query params to filter the view for a entities/groups belonging to a specific assignment. I'll defer to you!